types/split2/index.d.ts

```diff
@@ -21,7 +21,7 @@ declare namespace split {
     interface Options extends TransformOptions {
         maxLength?: number | undefined;
     }
-    type Matcher = string | RegExp;
+    type Matcher = string | RegExp | { [Symbol.split](string: string, limit?: number): string[]; };
```

Comment:
This type definition is copied from the original `String.split` declaration in lib.es2015.symbol.wellknown.d.ts of TS